fastlane/metadata/tr/subtitle.txt

```diff
@@ -1 +1 @@
-Bir web sitesi veya blog oluşturun
\ No newline at end of file
+Bir web sitesi/blog oluşturun
```

Comment:
Same note regarding the expected value and the upstream fix in GlotPress from the German version, with the only difference that, in this case, the Turkish translator updated my suggested translation to this version.

The `/` looks odd to me, but I know nothing about Turkish and trust the translator knows what they're doing 😄 Nevertheless, I sent a ping to the i18n about this, just to check and for my personal learning.